lib/Installation/AddOnProduct/AddOnProductPage.pm

```diff
@@ -22,7 +23,7 @@ sub init {
 
 sub is_shown {
     my ($self) = @_;
-    return $self->{rdb_specify_url}->exist();
+    return $self->{rdb_specify_url}->exist({timeout => 240});
```

Comment:
let's add a simple comment here about why we do this mentioning the System Probing screen, wdyt?

Reply:
comment added

Reply:
comment added